### DISCUSSION TITLE
Loki: Smarter query patterns with better suggestions

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#53444] [Observability]
Currently when we do query patterns suggestion, we have hardcoded values for parsers. It would be helpful if patterns were smarter and generated based on loki data (e.g. if log stream has logs in json format, suggest that as a parser)
<img width="1444" alt="image" src="https://user-images.githubusercontent.com/30407135/183615053-27a86c91-2615-4c1d-ab17-3d2b149f14d1.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
